MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
Should SDL2.dll be included in the DesktopGL template? (Same question for openal32.dll)

I know that it makes little sense because of the cross-platform nature of DesktopGL (and that it wouldn't solve OS X / Linux dependencies), but I'm wondering this to avoid misleading errors out-of-box on Windows.

@tomspilman ?